lib/portlayer/exec/base.go

```diff
@@ -304,16 +302,16 @@ func (c *containerBase) shutdown(ctx context.Context, waitTime *int32) error {
 
 	for _, sig := range stop {
 		msg := fmt.Sprintf("sending kill -%s %s", sig, c)
-		log.Info(msg)
+		op.Infof(msg)
 
-		timeout, cancel := context.WithTimeout(ctx, wait)
+		timeout, cancel := trace.WithTimeout(&op, wait, "shutdown")
 		defer cancel()
 
 		err := c.startGuestProgram(timeout, "kill", sig)
 		if err != nil {
 			// Just warn and proceed to waiting for power state per issue https://github.com/vmware/vic/issues/5803
 			// Description above in function kill()
-			log.Warnf("%s: %s", msg, err)
+			timeout.Warnf("%s: %s", msg, err)
```

Comment:
doesn't really matter...it will retain the opID since it was created via `trace.WithTimeout` -- but possibly more clear to use op...so will change to op